components/ProposalTopVotersNftChart.tsx

```diff
@@ -0,0 +1,82 @@
+import { useDigitalAssetsByOwner } from '@hooks/queries/digitalAssets'
+import { useNftRegistrarCollection } from '@hooks/useNftRegistrarCollection'
+import { PublicKey } from '@solana/web3.js'
+import { useMemo } from 'react'
+// import useVotePluginsClientStore from 'stores/useVotePluginsClientStore'
+
+interface Props {
+  showNfts: boolean
+  isNftMode: boolean
+  className?: string
+  voteType?: number
+  highlighted?: string
+}
+
+const getContents = (css, isLoading, highlighted, nfts, voteType) => {
```

Comment:
this function should not exist, nor should any similar function ever exist, the jsx should just be in the component